redecs_agent/redecs_agent.go

```diff
@@ -44,10 +133,24 @@ func logErrorNoFatal(err error) {
 var dockerClient *docker.Client
 var redisClient *redis.Client
 
-func reportServiceActive(serviceName string) {
+func reportServiceActive(serviceName string) error {
 	log.Debug("Reporting " + serviceName + " as active with IP " + configuration.LocalIp)
+	value := serviceName + "_" + configuration.LocalIp
 	// write {timestamp => SERVICENAME_IP} to ecssd:service_pings
-	redisClient.ZAdd("redecs:service_pings", redis.Z{float64(time.Now().Unix()), serviceName + "_" + configuration.LocalIp})
+	redisClient.ZAdd("redecs:service_pings", redis.Z{float64(time.Now().Unix()), value})
```

Comment:
`err := redisClient.Zadd(...).Err()`
 
https://godoc.org/gopkg.in/redis.v5#IntCmd.Err